File: Api/Domain/Models/Instructor.cs
namespace Domain.Models
{
    public class Instructor : BaseObject
    {
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string PhoneNumber { get; set; } = string.Empty;
        public IEnumerable<CourseAssignment> CourseAssignments { get; set; } = new List<CourseAssignment>();
    }
}

File: StaffFrontned/src/app/Domain/instructor.ts
export class Instructor {
  id: number;
  firstName: string;
  lastName: string;
  phoneNumber: string;

  constructor(
    id: number,
    firstName: string,
    lastName: string,
    phoneNumber: string
  ) {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.phoneNumber = phoneNumber;
  }
}

export class CreateInstructorDto {
  firstName: string;
  lastName: string;
  phoneNumber: string;

  constructor(firstName: string, lastName: string, phoneNumber: string) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.phoneNumber = phoneNumber;
  }
}

export class UpdateInstructorDto {
  id: number;
  firstName: string;
  lastName: string;
  phoneNumber: string;

  constructor(
    id: number,
    firstName: string,
    lastName: string,
    phoneNumber: string
  ) {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.phoneNumber = phoneNumber;
  }
}
